# HG changeset patch
# User Rik <rik@octave.org>
# Date 1362287607 28800
#      Sat Mar 02 21:13:27 2013 -0800
# Node ID 025393bef3996594954f648c6f64da2cecc70608
# Parent  3a93af21d2062d18d953c6ffc7f3f413b25d4461
range.tst: Correct typo min -> max in test.

* changed test/range.tst: Correct typo min -> max in test.

diff --git a/test/range.tst b/test/range.tst
--- a/test/range.tst
+++ b/test/range.tst
@@ -96,14 +96,14 @@
 %!assert (signbit (rrev([1 4 4 2])), logical ([0 1 1 0]))
 
 %!assert (min (r), -0)
 %!assert (signbit (min (r)), true)
 %!assert (min (rrev), -0)
 %!assert (signbit (min (rrev)), true)
 
 %!assert (max (rneg), -0)
-%!assert (signbit (min (rneg)), true)
+%!assert (signbit (max (rneg)), true)
 
 %!assert (sort (r, "descend"), [3 2 1 -0])
 %!assert (signbit (sort (r, "descend")), logical ([0 0 0 1]))
 %!assert (signbit (sort (rrev, "ascend")), logical ([1 0 0 0]))
 
